docker-compose.yml

```diff
@@ -135,7 +135,7 @@ services:
             discovery.type: single-node
             ES_JAVA_OPTS: -Xmx256m -Xms256m
         healthcheck:
-            test: ["CMD", "wget", "-O", "/dev/null", "--tries", "1", "localhost:9200/"]
+            test: ["CMD", "curl", "localhost:9200/_cluster/health?wait_for_status=green&timout=5s"]
             interval: 5s
```

Comment:
Does the interval timer start when the test command finishes, or does it run every 5s regardless of how long the health check takes?

Reply:
Also, typo in `timeout` option

Reply:
To quote from [docker healthcheck reference](https://docs.docker.com/engine/reference/builder/#healthcheck):
>The health check will first run interval seconds after the container is started, and then again interval seconds after each previous check completes.

I think the default should be ok so I'll remove the set interval.